schema/dataKeys.json

```diff
@@ -761,7 +761,11 @@
           "description": "Directory where RPM packages will be pushed"
         },
         "buildRpm": {
-          "type": "boolean",
+          "type": "string",
```

Comment:
I think we should use Boolean for this and not string. You just need to define it as Boolean in the RPA.